src/reference-manual/types.Rmd

```diff
@@ -60,14 +60,23 @@ section) can be made into an array type by declaring array arguments.
 For example,
 
 ```stan
-real x[10];
-matrix[3, 3] m[6, 7];
+array[10] real x;
+array[6,7] matrix[3, 3] m;
 ```
 
 declares `x` to be a one-dimensional array of size 10 containing
 real values, and declares `m` to be a two-dimensional array of
 size $6 \times 7$ containing values that are $3 \times 3$ matrices.
 
+Arrays can also be declared using the old array syntax. Equivalent
```

Comment:
Do something like:

```
Prior to 2.26 Stan models used a different syntax. Equivalent declarations of `x` and `m` are:
```

The thing that bothers me there is 'old' vs. 'new'. I think we should try to be specific about the times here -- like old and new in a year or so are gonna be kinda vague.

Reply:
Yeah, but I am not sure on what would be better names though?

Reply:
Maybe just call it the pre-2.26 array syntax. And then you can call the new stuff the syntax with the array keyword and we've gotten rid of the relative time terms

Reply:
Prior to 2.26 Stan models used a different syntax, with the array
dimensions defined after the variable identifier. 
Equivalent declarations of `x` and `m` with the pre-2.26 syntax:

```stan
real x[10];
matrix[3, 3] m[6, 7];
```

Reply:
That good ?

Reply:
Yup